Tests/Unit/LoRaTools/RegionAS923TestData.cs

```diff
@@ -68,9 +68,9 @@ public static IEnumerable<object[]> TestRegionFrequencyData()
                new object[] { region, 900, 8 },
                new object[] { region, 914.5, 9 },
                new object[] { region, 930, 10 },
-               new object[] { region, 923.4, 18 },
-               new object[] { region, 925.5, 90 },
-               new object[] { region, 923.2, 100 },
+               new object[] { region, 928.4, 18 },
```

Comment:
Can you elaborate on why we change the test cases?

Reply:
Before some of the test cases were actually using valid frequency for the region (e.g. AS923 has range 915 - 928 MHz) so e.g. 923.4 as we had before is a valid frequency. They were passing because previously we were checking dr and frequency together is `IsDataRateAndFrequencyValid()` so this would return false if only data rate was out of allowed range but not the frequency.